screens/QrLogin/QrLoginSuccessMessage.tsx

```diff
@@ -7,10 +7,17 @@ import { Centered, Button, Text, Column } from '../../components/ui';
 import { Theme } from '../../components/ui/styleUtils';
 import { useQrLogin } from './QrLoginController';
 import { QrLoginRef } from '../../machines/QrLoginMachine';
+import { APPLICATION_THEME } from 'react-native-dotenv';
 
 export const QrLoginSuccess: React.FC<QrLoginSuccessProps> = (props) => {
   const { t } = useTranslation('QrScreen');
   const controller = useQrLogin(props);
+  const clientName: Map<string, any> = new Map<string, any>([
+    ['purple', controller.clientName],
+    ['default', controller.clientName],
+    ['eldoria', 'Eldoria Authentication Platform'],
```

Comment:
Should we rely on clientName coming from esignet only?

Reply:
Are we getting clientName from the esignet?